frontend_tests/node_tests/settings_user_groups.js

```diff
@@ -260,4 +486,67 @@ set_global('people', {
         assert.equal(group_data.name, 'translated: mobile');
         assert.equal(group_data.description, 'translated: All mobile members');
     }());
+
+    (function test_user_groups_click_save_member_changes_triggered() {
+        var handler = $('#user-groups #1').get_on_handler("click", ".save-member-changes");
+        var realm_user_group = {
+            id: 1,
+            name: 'Mobile',
+            description: 'All mobile people',
+            members: [2, 4],
+        };
+        var fake_this = $.create('fake-#user-groups_click_save_member_changes');
+        user_groups.get_user_group_from_id = function (id) {
+            assert.equal(id, 1);
+            return realm_user_group;
+        };
+
+        channel.post = function (payload) {
```

Comment:
I think `payload` is a confusing variable name here, since really only `data` is part of the payload.  Can you rename it to `opts`?